CMakeLists.txt

```diff
@@ -170,8 +170,7 @@ configure_file(libnaboConfig.cmake.in
 # 2- installation build #
 
 # Change the library location for an install location
-get_filename_component(NABO_LIB_NAME ${libnabo_library} NAME)
-set(libnabo_library ${CMAKE_INSTALL_PREFIX}/lib/${NABO_LIB_NAME})
+set(libnabo_library ${LIB_NAME})
```

Comment:
Thanks. I've merged the urgent fix #65 already. It helps for the catkin projects. 
Maybe you can just test if that still works for you with arch. 

(
I believe to know why you did this: the generator expression you introduced for `libnabo_library` wasn't compatible with  `get_filename_component` because this would be applied to a not yet expanded generator expression. 
Using a similar generator expression that doesn't require the `get_filename_component` is also not enough because after it is installed the generator expression isn't longer valid due to the missing target. Hence, I hacked in the two steps in PR #65 .
)